docs/operator-manual/ui-extension/01-install.md

```diff
@@ -59,4 +63,40 @@ ___Policy Grid___
 
 ### Additional Features
 
-For installing additional features, follow the instructions in these docs to include [Metrics](./metrics.md) or [Tracing](./tracing.md).
+For installing additional features, follow the instructions in these docs to include [Metrics](./02-metrics.md) or [Tracing](./03-tracing.md).
+
+## Airgap Installation
+
+:::caution
+This requires Rancher Manager version `v2.8.0` or greater.
+:::
+
+As Kubewarden is considered a Rancher Official Extension, the Rancher team provides a mechanism to automatically generate an Extension Catalog Image that will be added to the `rancher-images.txt` file when [installing Rancher Manager](https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/publish-images#1-find-the-required-assets-for-your-rancher-version) for air-gapped instances.
```

Comment:
```suggestion
As Kubewarden is considered a Rancher Official Extension, the Rancher team provides a mechanism to automatically generate an Extension Catalog Image.
This will be added to the `rancher-images.txt` file when [installing Rancher Manager](https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/publish-images#1-find-the-required-assets-for-your-rancher-version) for air-gapped instances.
```